Headlines/src/DataSources/ProfileDataSource.swift

```diff
@@ -77,10 +77,10 @@ class ProfileDataSource: NSObject, UICollectionViewDataSource {
             return 0
         }
     }
-    
+
     func collectionView(_ collectionView: UICollectionView,
                         cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        
+
         switch indexPath.section {
         case 0:
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "interest_cell", for: indexPath) as? LabelCollectionViewCell else {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/adb6600e04f8d78e85d3ed821c2844be0fbd4321/Headlines/src/DataSources/ProfileDataSource.swift#L86"></span>Line should be 120 characters or less: currently 152 characters<br />
<code>line_length</code> <code>ProfileDataSource.swift:86</code></td>
    </tr>
  </tbody>
</table>